sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessMessageEventArgs.cs

```diff
@@ -31,6 +31,11 @@ public class ProcessMessageEventArgs : EventArgs
         /// </summary>
         public string EntityPath => _receiver.EntityPath;
 
+        /// <summary>
+        /// The identifier of the processor that raised this event.
+        /// </summary>
+        public string Identifier => _receiver.Identifier;
```

Comment:
I think we should probably use the Processor Identifier here.

Reply:
This also means that we will need a new ctor. Similar to the comment for the Error args, I think we should hide the old ctor.